res/css/structures/_UserMenuButton.scss

```diff
@@ -79,4 +98,49 @@ limitations under the License.
             justify-content: center;
         }
     }
+
+    .mx_IconizedContextMenu_icon {
+        position: relative;
+        width: 16px;
+        height: 16px;
+
+        &::before {
+            content: '';
+            width: 16px;
+            height: 16px;
+            position: absolute;
```

Comment:
Does this need to be `position: absolute` if it has no `top` / `left`?

Reply:
it needs something, but `display: block` is more sensible.